lib/init.js

```diff
@@ -0,0 +1,74 @@
+const EE = require('events').EventEmitter
+const { isMainThread } = require('./worker_threads')
+
+const initWorkers = require('./manager')
+const validateOpts = require('./validate')
+const noop = require('./noop')
+const runTracker = require('./runTracker')
+const track = require('./progressTracker')
+
+function init (opts, cb) {
+  return _init(opts, cb)
+}
+
+function _init (opts, cb, tracker) {
```

Comment:
It was like this [before](https://github.com/mcollina/autocannon/pull/298/files#diff-f922c690f6097716b39882a414117cba4684ab99dba74627fe8f82746ce0e5d3L51) as well.
And since it's to used [internally only](https://github.com/mcollina/autocannon/pull/298/files#diff-ab1cc8ca4845b72c91de0a25f68682c9514de0518a3985db531a854f02540024R25), I didn't change.

Reply:
Lol, ok. Let's fix this.